### ISSUE TITLE
Tags in discord posts

### ISSUE THREAD

**Issue by S100py** [OdysseyScorpio/FactionGist#4]
I'll be parsing (well trying to !) the output to tally up how much of each activity has been performed during each tick for MCRN. Carto posts currently have "in Desy" etc but missions have "System:x"
I'll use GSheets to search and extract the info. SO need to search for tags.
Can we standardise so we have something like "Mission System:x Faction:x Station:x" then DSystem:y DFaction:y DStation:y or similar for destinaitons (along with INF:x or Value:x etc) so I can easily find the info I want to tally against. We can also be brief and just state "Mission" "Carto" "Trade".  rather than "mission completed for" etc..

**Comment by OdysseyScorpio** [comment]
Yeah sure , it can be whatever you want, i have a config file for MCRN, i will send you it and you can make adjustments as you wish, i will let you know if i make any changes to align with the code . The aim is to have a way for each faction to edit their own config on the fly....but that might be a while away from having something automated
